markdown/preface.markdown

```diff
@@ -159,20 +160,24 @@ Although this book will focus on NumPy and SciPy,
 it is the many surrounding packages that make Python a powerhouse for
 scientific computing.
 
-
 ## The Great Cataclysm: Python 2 vs. Python 3
 
 In your Python travels, you may have already heard a few rumblings about which version of Python is better.
 You may have wondered why it's not just the latest version.
 
-At the end of 2008, the Python core developers released Python 3.0, a major update to Python with better Unicode (international) text handling, type consistency, and streaming data handling, among other improvements.
-To quote Douglas Adams in The Hitchhiker's Guide to the Galaxy, "this has made
-a lot of people very angry and been widely regarded as a bad move."
```

Comment:
The problem with the quote, at least from my perspective, is that a lot of people don't get it.  I know the quote by now, but it initially caught me by surprise and I thought that Python 3 was widely regarded as a flop.  I love jokes, but I think this one is too hard to pull off.